cypress/e2e/filling-form.cy.ts

```diff
@@ -0,0 +1,32 @@
+import {Login} from "../page/index-demoqa";
+import {PersonalInformation} from "../page/index-demoqa";
+
+const login = new Login();
+const personalinformation = new PersonalInformation();
+
+describe("Personal Information", () => {
+  it("then information is entered ", () => {
+    // Arrange
+    login.visitLoginPage();
+    // Action
+    const userdata = {name: "Holmes", lastName: "Salazar", email: "test@email.com", mobileNumber: "3656589156", currentAddress: "Av siempreViva # 123"};
+    personalinformation.fillForm(userdata);
+    personalinformation.genderData();
+    personalinformation.selectDayOfBirth();
+    personalinformation.selectMonth();
+    personalinformation.selectYear();
+    personalinformation.selectDay();
+    personalinformation.hobby();
+    personalinformation.otherHobby();
+    personalinformation.sub();
+    // Assertion
```

Comment:
This methods can be part of the "fillForm" method too and receive the values as parameters

Reply:
change made